### ISSUE TITLE
Use GtkFileChooserNative as file picker dialog

### ISSUE THREAD

**Issue by gudvinr** [wxWidgets/wxWidgets#24486]
### Description

With `GtkFileChooserNative` it is possible to override GTK+ dialog with `GTK_USE_PORTAL=1`, so applications will use native dialogs

That is probably only useful for KDE Plasma users though, but nice to have nonetheless.

**Comment by vadz** [comment]
Yes, this would be nice to have, any contributions would be welcome.

**Comment by FliegendeWurst** [comment]
I started working on this: https://github.com/FliegendeWurst/wxWidgets/commit/747bf40303d1c1f447554c1697173cc50fbd97b1

Currently it works well enough to use in PrusaSlicer. Did not yet test more advanced use cases. (And the code still throws dozens of *CRITICAL* Gtk/Glib warnings.)
